docs/04-reference/winglang-spec.md

```diff
@@ -216,6 +216,80 @@ it cannot be modified anymore.
 
 ---
 
+#### 1.1.5 `Duration`
+
+The `Duration` (alias `duration`) type represents a time duration.
+
+Duration literals are numbers with `m`, `s`, `h` suffixes:
```

Comment:
What about having a duration literal for every component of a datetime, for consistency:

```
year: Y;
month: M;
day: D;
hour: h;
min: m;
sec: s;
ms: ms;
```

Will probably also be useful for cron-like stuff later on.

Reply:
Wild! But maybe later?

Reply:
Very wild, although we should at least also have `ms` initially since that's pretty common/uncontroversial.